pkg/diagnostics/network/setup.go

```diff
@@ -177,15 +179,12 @@ func (d *NetworkDiagnostic) getNumOfValidTestPods(nsList []string, validPhases [
 		totalPodCount += len(podList.Items)
 
 		for _, pod := range podList.Items {
-			for _, phase := range validPhases {
-				if pod.Status.Phase == phase {
-					validPodCount += 1
-					break
-				}
+			if pod.Status.Phase == kapi.PodRunning {
+				runningPodCount += 1
```

Comment:
`} else { // build a list of pod state and any errors that can be retrieved for those that failed`